docs/handbook/04_streams.md

```diff
@@ -149,3 +149,71 @@ Of all the techniques that are included with Turbo, it's with Turbo Streams you'
 
 Turbo itself is completely backend-agnostic, though. So we encourage other frameworks in other ecosystems to look at the reference implementation provided for Rails to create their own tight integration.
+
+## Triggering Updates Using SSE and Mercure
+
+[The Mercure protocol](https://mercure.rocks) is a convenient way to trigger updates from any server application, even if it isn't able to maintain persistent WebSocket connections (ex: PHP).
+
+Mercure defines a simple interface to publish updates to a *hub*.
+When the *hub* receives an update, it broadcasts it using [Server-Sent Events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events) to every connected client. Several Open Source and commercial hubs [are available](https://mercure.rocks/spec#implementation-status).
+
+Connecting Turbo to a Mercure stream (or any other SSE stream) is straightforward, and doesn't require any external dependency:
+
+```javascript
+import { connectStreamSource } from "@hotwired/turbo";
+
+// The "topic" parameter can be any string or URI
+const es = new EventSource("https://example.com/.well-known/mercure?topic=my-stream");
+connectStreamSource(es);
+```
+
+To broadcast a Turbo Streams message, simply send a `POST` HTTP request to the Mercure hub:
+
+    curl \
+      -H 'Authorization: Bearer <snip>'
+      -d 'topic=my-stream' \
+      -d 'data=<turbo-stream action=...'
+      -X POST \
+      https://example.com/.well-known/mercure
+
+`topic` must be the same topic you subscribed in JavaScript, and `data` contains the Turbo Streams message.
+It's also possible to use [the other features supported by Mercure](https://mercure.rocks/spec#publication),
+including broadcasting private updates to authorized subscribers.
+
+Most Mercure *hubs* require the subscriber to be [authorized](https://mercure.rocks/spec#authorization). Refer to the documentation of your hub to learn how to generate a proper JSON Web Token.
+
+In this example, we use `curl` but any HTTP client will work.
+
+To disconnect the stream source, use the `disconnectStreamSource()` function:
+
+```javascript
+import { disconnectStreamSource } from "@hotwired/turbo";
+
+disconnectStreamSource(es);
+```
+
+In the following example, we use a Stimulus controller to connect to an SSE stream when an HTML element is created, and to disconnect when it is destroyed:
+
+```javascript
+// stream_controller.js
+import { Controller } from "stimulus";
+import { connectStreamSource, disconnectStreamSource } from "@hotwired/turbo";
+
+export default class extends Controller {
+  connect() {
+    this.es = new EventSource(this.element.getAttribute("data-url"));
```

Comment:
Maybe you could use the new values API as example: https://stimulus.hotwire.dev/reference/values?